src/applications/vre/28-1900/containers/OrientationWizardContainer.jsx

```diff
@@ -59,7 +58,7 @@ const OrientationWizardContainer = props => {
           can go directly to the online application without answering the
           questions below.{' '}
           <a
-            href={CHAPTER_31_ROOT_URL}
+            href="#"
```

Comment:
Josh asked me to keep this as a URL because sometimes users right/middle click to open the link in the new tab - see [526 `WizardContainer`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/all-claims/containers/WizardContainer.jsx#L55)